Gemfile

```diff
@@ -6,9 +6,7 @@ ruby '2.6.1'
 # Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', '>= 1.1.0', require: false
 gem 'govuk_elements_form_builder', '~> 1.3.0'
-gem 'govuk_elements_rails', '~> 3.1.3'
-gem 'govuk_template', '~> 0.25.0'
-gem 'gov_uk_date_fields', '~> 3.1.0'
+gem 'gov_uk_date_fields', git: 'git@github.com:ministryofjustice/gov_uk_date_fields.git', branch: 'govuk-frontend-v4'
```

Comment:
I am take it, that this will change  when this PR https://github.com/ministryofjustice/gov_uk_date_fields/pull/35 is merged.

Reply:
Yep! so once that other PR is reviewed and merged, we will release a new version of the gem, and use Rubygems instead of directly referencing the repo and branch here 👍